backend/package.json

```
@@ -9,7 +9,8 @@
     "test:coverage": "jest --coverage",
     "test:debug": "jest --detectOpenHandles",
     "lint": "eslint src/ --ext .jsx,.js,.ts,.tsx --max-warnings 0",
-    "lint:fix": "npm run lint -- --fix"
+    "lint:fix": "npm run lint -- --fix",
+    "createAPI": "node templates/create-api/index.js"
```

I wonder if we will have more of these functions... just my thought and totally fine with this.
```suggestion
    "create:API": "node templates/create-api/index.js"
```

---

that is a good point. maybe the migration and seed commands could go in here.